docs/portal.properties-Reference.md

```diff
@@ -412,12 +437,15 @@ To cache with Redis set `persistence.cache_type` to `redis`.
 To setup the Redis cache servers the following properties are required:
 
 ```
+redis.name=unique_cbioportal_instance_name
 redis.leader_address=redis://{host/servicename}:6379
 redis.follower_address=redis://{host/servicename}:6379
 redis.database=0
 redis.password=password
 ```
 
+The property `redis.name` will be used for distinguishing the caches among portal instances which might share their `app.name` value.
```

Comment:
```suggestion
If you are running one redis instance for multiple instances of cBioPortal, one can use the properties `redis.name` and `redis.database` to avoid clashes. If you are running only one instance of cBioPortal any value for name/database will do.
```